suite-native/app/e2e/tests/deeplinkPopup.test.ts

```diff
@@ -0,0 +1,115 @@
+import { exec } from 'child_process';
+// `expect` keyword is already used by jest.
+import { expect as detoxExpect } from 'detox';
+
+import { TrezorUserEnvLink } from '@trezor/trezor-user-env-link';
+import TrezorConnect from '@trezor/connect-deeplink';
+
+import { openApp } from '../utils';
+import { onOnboarding } from '../pageObjects/onboardingActions';
+import { onCoinEnablingInit } from '../pageObjects/coinEnablingActions';
+
+const TREZOR_DEVICE_LABEL = 'Trezor T - Tester';
+// Contains only one BTC account with a single transaction to make the discovery as fast as possible.
+const SIMPLE_SEED = 'immune enlist rule measure fan swarm mandate track point menu security fan';
```

Comment:
just recently, we started gathering all the mnemonics in one place here https://github.com/trezor/trezor-suite/blob/develop/packages/trezor-user-env-link/src/api.ts#L57 
not a required change, just a note to be considered

Reply:
Makes sense, I added it there https://github.com/trezor/trezor-suite/pull/14667/commits/17bb3c99871bb0ed3d0c7c9252459a9bf43bfdb0